scripts/api_dump_generator.py

```diff
@@ -434,6 +446,8 @@
 std::ostream& dump_text_{unName}(const {unName}& object, const ApiDumpSettings& settings, int indents);
 @end union
 
+//======================== pNext Chain Implementation =======================//
+
```

Comment:
Best to not include changes not related to what this pull request implements/fixes. Submit a separate pull request.